src/client/Streamer.ts

```diff
@@ -85,6 +103,10 @@ export class Streamer {
             });
             this._gatewayEmitter.on("VOICE_SERVER_UPDATE", (d) => {
                 if (guild_id !== d.guild_id) return;
+                
+                // channel_id is not set for guild voice calls
+                if(d.channel_id && (channel_id !== d.channel_id)) return;
+                
```

Comment:
`guild_id` is the one that's nullable, not `channel_id`. Is this correct?

Reply:
`guild_id` is null when it is a DM voice connection, and `channel_id` is missing (so undefined I guess?) from the gateway event for guild voice connections

Reply:
Okay got it (discord should really make it consistent...)

Reply:
.-.